mne/channels/montage.py

```diff
@@ -348,6 +348,60 @@ def get_positions(self):
         )
         return positions
 
+    @verbose
+    def add_estimated_fiducials(self, subject, subjects_dir=None,
+                                verbose=None):
+        """Estimate fiducials based on FreeSurfer ``fsaverage`` subject.
+
+        This takes a montage with the ``mri`` coordinate frame,
+        corresponding to the FreeSurfer RAS (xyz in the volume) T1w
+        image of the specific subject. It will call
+        :func:`mne.coreg.get_mni_fiducials` to estimate LPA, RPA and
```

Comment:
I don't like to have stuff from coreg module to be imported outside. coreg should be GUI code only I think.

Reply:
Agreed we could move it to `source_space.py`, it's where we deal with coordinate frames most of the time anyway